mindjunkies/forums/models.py

```
@@ -34,76 +39,76 @@ def save(self, *args, **kwargs):
     class Meta:
-        ordering = ["-created_at"]
+        ordering = ['created']
 
+class LikedReply(models.Model):
+    reply = models.ForeignKey(Reply, on_delete=models.CASCADE)
+    user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
+    created = models.DateTimeField(auto_now_add=True)
+    
     def __str__(self):
-        return f"Notification for {self.recipient.username}: {self.get_notification_type_display()}"
+        return f'{self.user.username} : {self.reply.body[:30]}'        
```

Ruff (0.8.2)

Trailing whitespace

Remove trailing whitespace

(W291)